mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -201,7 +202,7 @@ def Rock_ReduceOp :
 def Rock_ZeroInitKernelOp :
     Rock_Op<"zero_init_kernel", []>,
     Arguments<(ins AnyTensorOrMemRef:$buffer,
-                  StrAttr:$arch,
+                  Rock_GemmFeaturesAttr:$features,
```

Comment:
Also, I don't think this kernel needs the features, unless you're adding them to the gemm wrapper interface?

Reply:
Probably you are right... 

I just added features to every kernel that ends up using a buffer store which needs to know the features because from a layering perspective how buffer store is lowered kept independent of the kernel. However, since we know this kernel always want the buffer store lowered in a certain way that does not require features, we might as well put features::none for the buffer store.

Reply:
... Actually, good point on adding the features to the elementwise kernels, you might as well do it that way.